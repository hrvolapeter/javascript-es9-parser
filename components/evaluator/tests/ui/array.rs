var a = [1, "a", true];
var b = [a, 2];
console.log(a);
console.log(b);