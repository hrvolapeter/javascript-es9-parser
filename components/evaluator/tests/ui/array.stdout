{0: 1, 1: "a", 2: true}
{0: {0: 1, 1: "a", 2: true}, 1: 2}
